lib/ansible/modules/cloud/ovirt/ovirt_disk.py

```diff
@@ -536,6 +552,7 @@ def main():
         force=dict(default=False, type='bool'),
         sparsify=dict(default=None, type='bool'),
         openstack_volume_type=dict(default=None),
+        image_provider=dict(default=None),
```

Comment:
not a blocker but `default=None` is the default so it doesn't need to be specified.